capacities.go

```diff
@@ -0,0 +1,93 @@
+package packngo
+
+const capacityBasePath = "/capacity"
+
+// CapacityService interface defines available capacity methods
+type CapacityService interface {
+	List() (*CapacityReport, *Response, error)
+	Check(*CapacityInput) (*Response, error)
```

Comment:
I think that the Check can be Post (but don't dwell on it).

More importantly, I think it should return CapacityInput struct, indicating the availability in respective facilities. The returned JSON is (in case of insufficient capacity) the same as the JSON in the request, but with `"available":true/false`.

you could extend the ServerInfo with
```
Available *bool `json:"available,omitempty"`
```
and then the Available field could be unset in the request, and present in the respose.

This would need https://github.com/packethost/packngo/issues/96 to be fixed.